### PR TITLE
feat(vulnerability): add new result type

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/HAC-5787

## Description

Adds a field to Tekton Vulnerability scan

## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Screenshot 2024-07-23 at 3 38 53 PM
Screenshot 2024-07-23 at 3 43 19 PM

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/hooks/useScanResults.ts [hunk 1]
```diff
@@ -11,12 +11,14 @@ export const SCAN_RESULT = 'CLAIR_SCAN_RESULT';
...
 export const CVE_SCAN_RESULT = 'CVE_SCAN_RESULT';
 export const TEKTON_SCAN_RESULTS = 'TEKTON_SCAN_RESULTS';
+export const SCAN_OUTPUT = 'SCAN_OUTPUT';
 
 export const CVE_SCAN_RESULT_FIELDS = [
   SCAN_RESULT,
   SCAN_RESULTS,
   CVE_SCAN_RESULT,
   TEKTON_SCAN_RESULTS,
+  SCAN_OUTPUT,
```
Comment (sahil143) [inline src/hooks/useScanResults.ts:21]: you have not added `TEST_OUTPUT`. Do we not need it?

**Review by sahil143 — CHANGES_REQUESTED** [review]

File: src/hooks/useScanResults.ts (hunk 1 above)
Comment (abhinandan13jan) [reply src/hooks/useScanResults.ts:21]: I added and then removed it. Actually, we dont have a block for Test output. And this hook is related to scan results. 
Test output is carrying different details like success, warning and not the vulnerability data.

Comment (sahil143) [reply src/hooks/useScanResults.ts:21]: do we not use the `TEST_OUTPUT` data anywhere?

Comment (abhinandan13jan) [reply src/hooks/useScanResults.ts:21]: no not yet

**Review by sahil143 — APPROVED** [review]
/lgtm

**Pull request merged**